communications.md

```diff
@@ -2,95 +2,62 @@
 
 ## General guidelines
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+Communication is critical to any organization and in doing so, we should be considerate of other people's production processes and time.
+We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite, Asana and Emails.
 
-## First line of Communication: Texting
+Our communication tools encompasses both formal and informal communications as such we may want to understand:
 
-We use a few tools to help team members communicate with one another: Slack, G Suite and Asana. Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, use Slack, Gmail or Asana to contact another team member asynchronously (in an intermittent manner).
+- your audience 
+- the size of the audience 
+- content of message and reusability (if the content of the message will come in handy in the future) 
 
-Budget communication time. Try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
+and then choose the appropriate communication tools to ensure effective communication.
 
-#### - Internal & External Messaging Tool: Slack -
-Opt-in: Join channels that you are interested in or obligated to.
+Time management is key to success and so is effective communication. Sometimes communication tools can reduce productivity time hence, we can individually implement suitable patterns to ensure we effciently and effectively manage our time.
 
-Setting up Notifications: Once you're part of a channel, set up your notifications to suit your needs:
+Feel free to verbally ask colleagues what means they will like to be communicated through and utilize any of our communication tools with all professionalism.
```

Comment:
This defeats the purpose of async over sync

Reply:
I think Async-only is as harsh as a Sync-only policy. What's the middle ground here that optimized towards the most happiness with the least effect on productivity?

Idea: We outline some etiquette around in-person conversations/approaching someone and leave async as the go to for anything non-critical. This would be very educational and broadly applicable across all roles within the company.

Reply:
I think one of our strength is encourage members to jump on task they are interest in, this gives member motivation to do a better job than just complete an assigned task. We should prevent the case which some member missed out a task they interested in just because he was not involved in the conversation. 

I agree that in person conversation can get point across faster and easier, but with fluid team structure this greatly limits other team member’s chance to contribute in the subject, while async type such as communicate through slack opens a channel for rest of team to join immediately or later at their convene regardless their physical location or time. 

I also kind worried about without a baseline type of communication, will this create a artificial group based on this preference. people may tends to communicate more with the persons that has same preferred way of communication.

So I suggest we should recommend an async way of communication as a base line, this doesn’t mean member can’t have a direct conversion, but rather to be educated through the document that when choosing how to communicate, been in respect of whole team, does this involve only the ones you are going to talk to or others may also contribute in it. does this require to be referenced in a later date, if so then maybe some text message is easier the recall why you arrived to certain conclusion. If all above are not concern and other persons are free to have the direct conversation, then by all means have a in person conversation.

Reply:
> I think one of our strength is encourage members to jump on task they are interest in, this gives member motivation to do a better job than just complete an assigned task. We should prevent the case which some member missed out a task they interested in just because he was not involved in the conversation.

@steven-mindswire this is true and it never changed. Below is the actual statement that is written. (attention: to the last statement)

>**- Project Management & Product Issue Documentation: Asana -**
Our Asana workspace is commonly used for project management e.g task allocation, task state (in progress or completed), and any other process we deem it fit to be used for (e.g Office Availability, calender scheduling, etc.) Opt-in to join projects/tasks that you are interested in or obligated to.

Reply:
> I agree that in person conversation can get point across faster and easier, but with fluid team structure this greatly limits other team member’s chance to contribute in the subject, while async type such as communicate through slack opens a channel for rest of team to join immediately or later at their convene regardless their physical location or time.

@steven-mindswire This is also true and I believe a statement in this document takes care of that (not explicitly stated though). See below 
>Our communication tools encompasses both formal and informal communications as such we may want to understand:
>- your audience
>- the size of the audience
>- content of message and reusability (if the content of the message will come in handy in the future)

>and then choose the appropriate communication tools to ensure effective communication.

Reply:
> I also kind worried about without a baseline type of communication, will this create a artificial group based on this preference. people may tends to communicate more with the persons that has same preferred way of communication.

I do really understand your concern about this. I think communication in a business space is somewhat different. Reason is because communication here can be mostly based on the job and thus, if your preferred communication differs from another it does not really hinder it as much. We can all be communicating async and still encounter similar problem because some people will feel others are more approachable. Regardless your point is noted and we should find a way around it.

There are pros and cons with both async and sync. depending on the use case, any can be the preferred/best option. I agree with having this on our slack profile and we see how it plays out.

**Extra: On a lighter note**
I just want to add that what we do is scary enough (lol) and when we over stressing communication  like this, new staff might become overly cautious and have the misconception that we also track staff communication content. :)

Reply:
> - Project Management & Product Issue Documentation: Asana -
Our Asana workspace is commonly used for project management e.g task allocation, task state (in progress or completed), and any other process we deem it fit to be used for (e.g Office Availability, calender scheduling, etc.) Opt-in to join projects/tasks that you are interested in or obligated to.

It is a way of join the project, but shouldn't be the only way. One person may not be interested in a project initially, but rather through the reading the note and conversations about the project then realize there are something they can are interested to contribute, we need to understand open conversation can be an way of inspiration.

> Our communication tools encompasses both formal and informal communications as such we may want to understand:
your audience
the size of the audience
content of message and reusability (if the content of the message will come in handy in the future)
and then choose the appropriate communication tools to ensure effective communication.

the problem is how to identify the correct audience, the purpose of this guid line is to tell there are more person may involved than you think, and when there is doubt, use the slack. simply saying you should understand your audience and make a decision is way too empty and doesn't really contribute the effective we are aiming to achieve 

> I do really understand your concern about this. I think communication in a business space is somewhat different. Reason is because communication here can be mostly based on the job and thus, if your preferred communication differs from another it does not really hinder it as much. We can all be communicating async and still encounter similar problem because some people will feel others are more approachable. Regardless your point is noted and we should find a way around it.

in a more rigid team structure, yes, the communication is mostly happened within the group you worked with, so you know very well who is involved, their preference and what they are working on. This become difficult to track when we try to keep project open to the whole team, so boundary on who is involved in the "job" become less clear and you may not talking to all the person that potentially should be involved as you thought. Other issue is by the modularized natural of our product, so when you communicate some issue that affects your components, there are good chance the issue is originate from way upper stream or even from other team's components. so either you understand perfect where the problem is and able locate the correct personal, or you going to waste a lot of time to repeat your question and drag half of the team on it. 

> Extra: On a lighter note
I just want to add that, what we do is scary enough (lol) and when we over stress communication, new staff might become overly cautious and have the misconception that we also track staff communication content. :)

nothing to do with what we do, but team does function more effectively when communication is good. since we can't avoid grow and we want to keep the flexible and openness as a small team, some set of guideline is necessary. these guidelines is not simply a set of rule we recommend team member to follow, rather it explain the problem and thought process when we come up these guidelines , so when in practice, any issue can be trace back and improved. 

@odesiuz

Reply:
> It is a way of join the project, but shouldn't be the only way. One person may not be interested in a project initially, but rather through the reading the note and conversations about the project then realize there are something they can are interested to contribute, we need to understand open conversation can be an way of inspiration.

@steven-mindswire  sure thing. We can expand on that point for clarity.

Reply:
> the problem is how to identify the correct audience, the purpose of this guid line is to tell there are more person may involved than you think, and when there is doubt, use the slack. simply saying you should understand your audience and make a decision is way too empty and doesn't really contribute the effective we are aiming to achieve

@steven-mindswire sure. identifying the audience is always an issue and there is a phrase you made that can give more clarity without turning the guideline into an instruction. The phrase is (drumroll!!) 
> and when there is doubt, use the slack.   👍 

A simple phrase like this can encourage (maybe "default") people to using slack without seeing the guideline as an instruction or specifically instructing them to "default" to one of async or sync. I feel this phrase will be a great addition to that portion of the document to give it a clearer meaning especially when everyone, some or none is an expert in identifying the audience.

"The values [of a company] are its North Star" but not an instruction manual" by Foehl, Robert
Executive in Residence for Business Law and Ethics Ohio University

Reply:
> nothing to do with what we do, but team does function more effectively when communication is good. since we can't avoid grow and we want to keep the flexible and openness as a small team, some set of guideline is necessary. these guidelines is not simply a set of rule we recommend team member to follow, rather it explain the problem and thought process when we come up these guidelines , so when in practice, any issue can be trace back and improved.

@steven-mindswire True it should explain problems and thought process. Let's keep the reviews coming if need be and I am sure by the end we will have a well worded document that better suits our purpose and clearly reflects our values.

Reply:
> @steven-mindswire sure. identifying the audience is always an issue and there is a phrase you made that can give more clarity without turning the guideline into an instruction. The phrase is (drumroll!!)
and when there is doubt, use the slack. 👍
A simple phrase like this can encourage (maybe "default") people to using slack without seeing the guideline as an instruction or specifically instructing them to "default" to one of async or sync. I feel this phrase will be a great addition to that portion of the document to give it a clearer meaning especially when everyone, some or none is an expert in identifying the audience.
"The values [of a company] are its North Star" but not an instruction manual" by Foehl, Robert Executive in Residence for Business Law and Ethics Ohio University

I think this is exactly what we are trying to avoid in the document, to give a single line and ask people to follow it blindly, rather, we should look at this more like a propose to change. 

As we mentioned above, there is no enforcement of these set of guideline, rather with the reason stated in the document, when there is situation, maybe try out the new way of communication that potentially improve the efficiency of communication when you dealing with a larger team. if you find some part doesn't work for you or you think there is something need to be changed or added, you can always put your scenario into this. So, to start this dynamic and organic growing "guideline", putting down scenario with proposed solution is essential, after all, you can't convince some one to try something new willingly if you don't explain the reason. 

“You never change things by fighting the existing reality. To change something, build a new model that makes the existing model obsolete.” by Buckminster Fuller,American architect, systems theorist, author, designer, inventor, and futurist.